src/inc/shash.h

```diff
@@ -479,6 +498,10 @@ class SHash : public TRAITS
             return *(const Iterator*)this;
         }
 
+        KeyIterator()
```

Comment:
Thanks for pointing this out, I believe I no longer have a dependency on the edits in this file and I can back them out. If I remember right I did something like this in an earlier version of the PR that made a default constuctor useful:
KeyIterator foo;
if(x) { foo = bar } else {foo = baz}. 



---
In reply to: [127312711](https://github.com/dotnet/coreclr/pull/12193#discussion_r127312711) [](ancestors = 127312711)